corehq/apps/hqmedia/tasks.py

```diff
@@ -186,10 +187,24 @@ def build_application_zip(include_multimedia_files, include_index_files, app,
                         file_cache[path] = data
 
         if toggles.LOCALE_ID_INTEGRITY.enabled(app.domain):
-            errors.extend(find_missing_locale_ids_in_ccz(file_cache))
+            locale_errors = find_missing_locale_ids_in_ccz(file_cache)
+            if locale_errors:
+                errors.extend(locale_errors)
+                notify_exception(
+                    None,
+                    message="CCZ missing locale ids from default/app_strings.txt",
```

Comment:
nit: i think this would end up as the subject.
How about adding the environment here [icds] or [india] would be helpful to know where is it coming from.

Reply:
It might make sense to add environment as a prefix within `notify_exception` itself, to all emails, but I'm going to call that out of scope.

Reply:
that works too